src/pages/SignupPage.js

```diff
@@ -17,9 +18,14 @@ const SignupPage = () => {
   const [passwordError, setPasswordError] = useState(false);
   const [errorMessage, setErrorMessage] = useState(undefined);
   const [isLoading, setIsLoading] = useState(false);
+  const [conditionsAccepted, setConditionsAccepted] = useState(false);
   const navigate = useNavigate();
   const theme = useTheme();
 
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
```

Comment:
I don't think it's necessary to create a function just to setOpen(true) or setOpen(false)

Reply:
It was like that in the Material UI documentation. i just copied. But you probably right.